test/specs/entityVisibilityEvents.js

```diff
@@ -0,0 +1,55 @@
+import { expect } from "chai";
+import selectors from "./../../data/selectors.json";
+import expected from "./../../data/expected.json";
+import testData from "./../../data/testData.json";
+import {recordCreationInit,
+    recordCreationInitNoInput,
+    clickActionDelete,
+    reset,
+    userLogin} from "../../helper/methods";
+
+describe('Visibility Events Page', () => {
+    before( () => {
+        browser.maximizeWindow();
+        browser.url('https://ref2.eteam.work/');
+        userLogin(testData.userLogin.loginName250,
+            testData.userLogin.password250);
+        reset();
+        $(selectors.visibilityEvents.visEventsBnt).click();
+        expect($(selectors.visibilityEvents.visEventsBarTitle).getText())
+            .to.equal(expected.visibilityEvents.visEventsText);
```

Comment:
Create a test for these statements
```suggestion
     it('Should verify that...', () => {
        $(selectors.visibilityEvents.visEventsBnt).click();
        expect($(selectors.visibilityEvents.visEventsBarTitle).getText())
            .to.equal(expected.visibilityEvents.visEventsText);
     });
```